rocks-lib/src/lua_package/mod.rs

```diff
@@ -40,15 +40,15 @@ impl LuaPackage {
 /// A lua package requirement with a name and an optional version requirement.
 #[derive(Debug, Clone, PartialEq)]
 #[cfg_attr(feature = "clap", derive(clap::Args))]
-pub struct LuaPackageReq {
+pub struct PackageReq {
```

Comment:
Might seem like an odd rename, but I found `LuaPackageReq` to miss the point a bit. It's not a requirement on a `LuaPackage`, since those are always physical rocks in a rocktree.

In our code, we use this struct as general requirement matching on any sort of package, so I just called it `PackageReq`.